src/screens/ContactScreen/elements/ItemUser.js

```diff
@@ -60,14 +66,22 @@ const ItemUser = ({photo, username, followed, onPress, userid}) => {
   );
 };
 
+ItemUser.propTypes = {
+  photo: PropTypes.string,
+  username: PropTypes.string,
+  followed: PropTypes.any,
+  onPress: PropTypes.func,
+  userid: PropTypes.string,
+  isAnon: PropTypes.string
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's great to see PropTypes being used for type checking in React. However, the `isAnon` prop is defined as a string here, but it seems like it should be a boolean based on its usage in the component. Please correct this if it's an error.
```diff
-  isAnon: PropTypes.string
+  isAnon: PropTypes.bool
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->